src/plat/rk3288/config.cmake

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+#
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+cmake_minimum_required(VERSION 3.7.2)
+
+# We introduce a variable to hold this long expression to prevent the
+# code styler from line-wrapping the declare_platform() statement.  We
+# want to keep that on one line so the `griddle` tool (or humans) can
+# easily `grep` a list of supported platforms.
+# For now we don't support HYP... but include it for later.
+set(AArch32OrArchArmHyp "KernelSel4ArchAarch32 OR KernelSel4ArchArmHyp")
+declare_platform(rk3288 KernelPlatformRK3288 PLAT_RK3288 ${AArch32OrArchArmHyp})
+unset(${AArch32OrArchArmHyp} CACHE)
+
+set(cmake_configs KernelPlatformRK3288NTablet)
+set(c_configs PLAT_RK3288_NTABLET)
+set(plat_lists rk3288-ntablet)
```

Comment:
This CMake config pattern shouldn't be needed here.  It comes from (https://github.com/seL4/seL4/blob/master/src/plat/exynos5/config.cmake#L18) where there are sub-platforms within the exynos5 platform. We try to avoid sub-platforms and just have a flat namespace for all platform variants the kernel supports under the KernelPlatform setting. https://github.com/seL4/seL4/blob/master/src/plat/imx8m-evk/config.cmake is an example of this.

Reply:
Cool.  I'l try to fix it ...

Reply:
I've pushed some changes to the pull request here and in seL4-tools to address this.